index.js

```diff
@@ -13,10 +13,16 @@ module.exports = {
 	},
 	extends: ['eslint:recommended', 'plugin:react/recommended', 'plugin:react-hooks/recommended'],
 	ignorePatterns: [
-		'**/node_modules/**',
-		'**/build/**',
-		'**/dist/**',
-		'**/coverage/**'
+		'/**/node_modules/',
+		'/**/node_modules/*',
+		'/**/build/',
+		'/**/build/*',
+		'/**/dist/',
+		'/**/dist/*',
+		'/**/coverage/',
+		'/**/coverage/*',
+		'/**/resources/',
+		'/**/resources/*'
```

Comment:
there's some variations between versions of eslint from what I've observed. This particular combinations cover them all, and plus, in a way that handles ignores the directories entirely, rather than scanning everything filtering out the file results.  It was the fasted combination I observed.